src/api/searchApi.ts

```diff
@@ -238,31 +238,39 @@ export enum NviCandidatesSearchParam {
   Aggregation = 'aggregation',
   Assignee = 'assignee',
   ExcludeSubUnits = 'excludeSubUnits',
+  ExcludeUnassigned = 'excludeUnassigned',
+  GlobalStatus = 'globalStatus',
   Filter = 'filter',
   Offset = 'offset',
   OrderBy = 'orderBy',
   Query = 'query',
   Size = 'size',
+  Status = 'status',
   SortOrder = 'sortOrder',
   Year = 'year',
-  Visibility = 'visibility',
 }
 
 export type NviCandidateOrderBy = 'createdDate';
 
+export type NviCandidateFilter = 'rejectedByOthers' | 'approvedByOthers' | 'collaboration';
+export type NviCandidateStatus = 'pending' | 'approved' | 'rejected';
```

Comment:
sjekk denne opp mot typen som ligger i `nvi.types.ts`

Reply:
ser ut som begge trengs inntil videre, da den i `nvi.types.ts` bl.a. brukes for å vise status for hver enkelt kandidat som chip i søketreff